code/controllers/subsystems/records.dm

```diff
@@ -179,21 +179,42 @@
 /datum/controller/subsystem/records/proc/reset_manifest()
 	manifest.Cut()
 
-/datum/controller/subsystem/records/proc/get_manifest(var/monochrome = 0, var/OOC = 0)
+// The one and only method for showing a pop-up crew manifest (browser) window
+/proc/open_crew_manifest(var/mob/user, var/OOC = FALSE)
+	if(!user)
+		return
+	var/const/windowname = "manifest"
+	var/dat = {"<h2 style="text-align: center">Crew Manifest</h2>"}
+	dat += SSrecords.get_manifest(monochrome = FALSE, OOC = OOC)
+	send_theme_resources(user)
+	user << browse(enable_ui_theme(user, dat), "window=[windowname];size=450x600")
+	return windowname
+
+/datum/controller/subsystem/records/proc/get_manifest(var/monochrome = FALSE, var/OOC = FALSE)
```

Comment:
The monochrome variable is pretty pointless. I'd just axe it

Reply:
That's fair; I didn't want to try to decipher the list manifest format but it turns out it's not too bad. Removed.

I added a simple, short proc for generating the text for printing instead.